src/event/v1/cloudevent.ts

```diff
@@ -58,7 +58,9 @@ export interface CloudEventV1Attributes {
    * @example com.example.object.delete.v2
    */
   type: string;
+}
 
+export interface CloudEventV1OptionsAttributes {
```

Comment:
Can you name this `CloudEventV1OptionalAttributes` instead?

Reply:
right,  that name makes more sense